src/topology/continuous_function/compact.lean

```diff
@@ -394,4 +423,28 @@ end
 
 end comp_right
 
+section weierstrass
+variables {X : Type*} [topological_space X] [t2_space X] [locally_compact_space X]
+variables {E : Type*} [normed_group E] [complete_space E]
+
+/-- **Weierstrass M-test** -/
+lemma summable_of_locally_summable_norm {ι : Type*} {F : ι → C(X, E)}
+  (hF : ∀ K : set X, compact_space K → by exactI summable (λ i, ∥(F i).restrict K∥)) :
+  summable F :=
+begin
+  classical,
+  change ∃ _, tendsto _ _ _,
```

Comment:
Does this indicate some missing API?

Reply:
I replaced by `simp only [summable, has_sum]`.  (You might be suggesting we introduce lemmas `summable_def`, `has_sum_def`, but I believe there's a school of thought that we should avoid such lemmas unless the desired form is syntactically (or definitionally) different.)